rfid_tag_finder:
  ros__parameters:

    # ROS Parameters
    map_frame: map
    odom_frame: odom
    laser_link_frame: base_laser_front_link
    real_laser_link_frame: real_laser_link
    scan_topic: scan
    rfid_tag_poses_topic: rfid_tag_poses
    odom_topic: odom
    cmd_vel_topic: cmd_vel
    initial_pose_topic: initialpose
    real_laser_link_pose_topic: real_laser_link_pose

    # other parameters
    laserscan_noise_stddev: 0.01
    odom_noise_linear_stddev: 0.01
    odom_noise_angular_stddev: 0.01   
    
    rfid_tags:
      A: [1.0, 1.0]
      B: [6.0, 1.0]
      C: [3.0, -1.0]
      D: [1.0, -3.0]
      E: [4.0, -4.0]


localisation_using_kalman:
  ros__parameters:

    # ROS Parameters
    map_frame: map
    odom_frame: odom
    laser_link_frame: base_laser_front_link
    real_laser_link_frame: real_laser_link
    scan_topic: scan
    rfid_tag_poses_topic: rfid_tag_poses
    odom_topic: odom
    initial_pose_topic: initialpose
    real_laser_link_pose_topic: real_laser_link_pose
    estimated_base_link_pose_topic: estimated_base_link_pose

    # other parameters
    minimum_travel_distance: 0.1
    minimum_travel_heading: 0.1

    rfid_tags:
      A: [1.0, 1.0]
      B: [6.0, 1.0]
      C: [3.0, -1.0]
      D: [1.0, -3.0]
      E: [4.0, -4.0]    
